inc/post-type/edit.php

```diff
@@ -327,6 +327,8 @@ public function register_meta_boxes( $meta_boxes ) {
 			),
 		);
 
+		$advanced_fields = apply_filters( 'rwmb_advanced_field_settings', $advanced_fields, $label_prefix, $args_prefix );
```

Comment:
Can you change the hook to `mbcpt_advanced_fields`? That sounds more convenient.

Reply:
yeah that is actually better